cross/pip/Makefile

```diff
@@ -1,9 +1,10 @@
 PKG_NAME = pip
-PKG_VERS = 9.0.1
+PKG_VERS = 18.0
```

Comment:
I do not understand how pip upgrade is related to GateOne. If a new version is required, I guess Python has to be updated to include newer pip version, right?

Reply:
They changed to a year-based version scheme, but actually the code changes are super minor.